src/test/java/htsjdk/samtools/cram/structure/ContainerTest.java

```diff
@@ -171,15 +171,16 @@ public static void distributeIndexingParametersToSlicesOneSlice() {
         final int containerStreamByteOffset = 100000;
 
         // this Container consists of:
-        // a header of size 1234 bytes
+        // a header (size irrelevant)
+        // a Compression Header of size 7552 bytes
         // a Slice of size 6262 bytes
 
-        final int containerHeaderSize = 1234;
+        final int compressionHeaderSize = 7552;
```

Comment:
the meaningful changes here are below, in `createOneSliceContainer()` / `createTwoSliceContainer()`